<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Digital Holographic Microscope</string>
  </property>
  <property name="locale">
   <locale language="English" country="Canada"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>25</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 25pt &quot;Arial Black&quot;;


</string>
    </property>
    <property name="text">
     <string>Digital Holographic Microscope</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="version">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>V 0.0.1, April, 2022</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="standard_TDHM">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>3D Image with Single wavelenth</string>
    </property>
    <property name="statusTip">
     <string>3D Image with Single wavelenth</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 87 10pt &quot;Arial Black&quot;;
</string>
    </property>
    <property name="text">
     <string>Standard 
TDHM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="in_line_TDHM">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>3D cell/particle tracking (please block the reference beam)</string>
    </property>
    <property name="statusTip">
     <string>3D cell/particle tracking (please block the reference beam)</string>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 87 10pt &quot;Arial Black&quot;;
</string>
    </property>
    <property name="text">
     <string>3D tracking
In-line TDHM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Standard TDHM with two wavelenth illumination</string>
    </property>
    <property name="statusTip">
     <string>Standard TDHM with two wavelenth illumination</string>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 87 10pt &quot;Arial Black&quot;;
</string>
    </property>
    <property name="text">
     <string>Dual-wave
TDHM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;
</string>
    </property>
    <property name="text">
     <string>Comming</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 bold 26pt &quot;Calibri&quot;;
</string>
    </property>
    <property name="text">
     <string>Yip Lab</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 220, 220);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>icons/UofT_label.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFilt">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFilt"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons/icon.qrc"/>
 </resources>
 <connections/>
</ui>
